tests/bsim/bluetooth/host/misc/hfc/src/utils.h

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright (c) 2023 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include "bs_tracing.h"
+#include "bs_types.h"
+#include "bstests.h"
+
+#define BS_SECONDS(dur_sec)    ((bs_time_t)dur_sec * USEC_PER_SEC)
+#define TEST_TIMEOUT_SIMULATED BS_SECONDS(60)
+
+extern enum bst_result_t bst_result;
+
+#define DECLARE_FLAG(flag) extern atomic_t flag
+#define DEFINE_FLAG(flag)  atomic_t flag = (atomic_t) false
+#define SET_FLAG(flag)     (void)atomic_set(&flag, (atomic_t) true)
+#define UNSET_FLAG(flag)   (void)atomic_set(&flag, (atomic_t) false)
```

Comment:
```suggestion
#define DEFINE_FLAG(flag)  atomic_t flag = (atomic_t)false
#define SET_FLAG(flag)     (void)atomic_set(&flag, (atomic_t)true)
#define UNSET_FLAG(flag)   (void)atomic_set(&flag, (atomic_t)false)
```

IIRC we don't have spaces between values and the cast